src/styles/components/_trending.scss

```diff
@@ -0,0 +1,42 @@
+.trending {
+  max-width: 30%;
+  flex: 1 1 auto;
+
+  .article-list{
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    overflow: scroll;
```

Comment:
I think you just want scroll-y in this case. Definitely don't want two way scroll.